sdk/mediaservices/Azure.ResourceManager.Media/api/Azure.ResourceManager.Media.netstandard2.0.cs

```diff
@@ -1954,7 +1954,7 @@ public partial class LiveEventInput
         public string AccessToken { get { throw null; } set { } }
         public System.Collections.Generic.IList<Azure.ResourceManager.Media.Models.LiveEventEndpoint> Endpoints { get { throw null; } }
         public System.Collections.Generic.IList<Azure.ResourceManager.Media.Models.IPRange> IPAllowedIPs { get { throw null; } }
-        public System.TimeSpan? KeyFrameIntervalDuration { get { throw null; } set { } }
+        public string KeyFrameInterval { get { throw null; } set { } }
```

Comment:
according to the example, the value should be `duration`, but here service return an empty string, so would change the generated code to use `TimeSpan.TryParse` instead of `TimeSpan.Parse`